plugins/eni/commands/commands.go

```diff
@@ -66,23 +66,24 @@ func add(args *skel.CmdArgs, engine engine.Engine, dhclient engine.DHClient) err
 	if err != nil {
 		return err
 	}
+	log.Infof("Found ENI with mac address (id=%s): %s", conf.ENIID, macAddressOfENI)
```

Comment:
This message is very similar to the one below, can you add context eg: `from metedata` or `on the host` separately into those logs?